main.go

```diff
@@ -108,7 +109,10 @@ func main() {
 	}
 
 	if versionFlag {
-		fmt.Printf("commit=%s, build date=%s, build source=%s, version=%s, os=%s, arch=%s\n", commit, date, buildSource, version, runtime.GOOS, runtime.GOARCH)
+		cmd := secureexec.Command("git", "--version")
```

Comment:
Thanks for raising a PR :)

Three things:
1) you'll need to rebase onto master because this code now lives in at `pkg/app/entry_point.go:80`. If you get merge conflicts you may be better off just copying this chunk of newly added code, hard-resetting onto master, and then pasting it in again
2) can we extract out a `getGitVersionInfo() string` function to use here?
3) in Go, we opt for using camelCase as opposed to snake_case in variable names. So git_version should be gitVersion